manifests/install.pp

```diff
@@ -46,8 +46,9 @@
   # npm
   if $nodejs::npm_package_name and $nodejs::npm_package_name != false {
     package { $nodejs::npm_package_name:
-      ensure => $nodejs::npm_package_ensure,
-      tag    => 'nodesource_repo',
+      ensure        => $nodejs::npm_package_ensure,
+      allow_virtual => false,
```

Comment:
I've debated whether this should be `allow_virtual => $nodejs::npm_package_ensure != 'absent'` or even

```puppet
if $nodejs::npm_package_ensure == absent {
  $allow_virtual = false
} else {
  $allow_virtual = undef
}
```

WDYT?

Reply:
Or, even stronger, add `manage_package_repo` to the mix, as we have no friggin idea how other packages might be crazy.